apps/scan/backend/package.json

```diff
@@ -70,7 +70,7 @@
     "rxjs": "7.8.1",
     "tmp": "^0.2.1",
     "uuid": "9.0.1",
-    "xstate": "^4.32.1",
+    "xstate": "^4.33.0",
```

Comment:
Upgraded xstate to resolve a specific bug where a delayed event from a child machine wasn't getting canceled after the child machine was stopped, which caused a warning to be logged every time we stopped polling